eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -144,6 +144,24 @@ stages:
       ### Additional jobs for full build ###
       - ${{ if in(parameters.scope, 'full') }}:
 
+        # This AlmaLinux leg is intended to build with the min supported glibc version
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+            buildName: ${{ format('{0}_Online_MsftSdk', variables.almaLinuxName) }}
+            isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+            vmrBranch: ${{ variables.VmrBranch }}
+            architecture: x64
+            pool: ${{ parameters.pool_Linux }}
+            container: ${{ variables.almaLinuxContainer }}
+            buildFromArchive: false            # 🚫
+            buildSourceOnly: true              # ✅
+            enablePoison: false                # 🚫
+            excludeOmniSharpTests: true        # ✅
```

Comment:
Is there a specific reason for excluding these tests?

Reply:
Because we don't ship on alma linux and its intent is to validate glibc compatibility.